### ISSUE TITLE
List of figures does not work when pictures in hidden table

### ISSUE THREAD

**Issue by anneloredev** [Alwinator/asciidoctor-lists#12]
I have some pictures I want to display side by side in a pdf. The easiest way to do this is by putting them side-by-side inside a hidden table. Images, which are inside the hidden table do not appear in the list of figures.

I assume that it is similar to [this bug I found in asciidoctor-bibtex](https://github.com/asciidoctor/asciidoctor-bibtex/issues/85), where the issue is that the extension does not traverse AsciiDoc table cells.

**Comment by Alwinator** [comment]
@anneloreegger Thank you for your bug report. Please add an asciidoctor code for reproduction.

**Comment by anneloredev** [comment]
@Alwinator Here is the  example asciidoctor code:

```adoc
= Test project
Jane Doe
25.05.2022
:organization: Company Name
:doctype: book
// Settings:
:experimental:
:reproducible:
:icons: font
:listing-caption: Listing
:toc: macro
:toclevels: 3

[discrete]
== Abstract
This is the abstract of our project

toc::[]

== List of Figures
list-of::image[]

:sectnums:

.Route
image::include/runkeeper_route.png[]

[cols="1,1",grid=none,frame=none]
|===
a|.Route
image::include/runkeeper_route.png[]
a|.Runkeeper Training and Insight 
image::include/runkeeper_training_insight.png[]
|===
```

The first image I have put appears in the list of figures. The second and third images I have put (inside the hidden table) do not appear in the list of figures.

Let me know if you need more information to reproduce the bug.

**Comment by Alwinator** [comment]
Fixed in version [v1.0.8](https://github.com/Alwinator/asciidoctor-lists/releases/tag/v1.0.8).
